modules/islandora_crossref_citations/islandora_crossref_citations.module

```diff
@@ -71,12 +71,9 @@ function islandora_crossref_citations_block_view($delta = '') {
       $badge = $block_text . ': ' . $result_citation_count;
     }
     else {
-      $badge = '<img src="https://img.shields.io/badge/' . $block_text . '-' . $result_citation_count . '-blue.svg?style=flat" alt="' . t('Number of citations via Crossref: ' . $result_citation_count) . '">';
+      $badge = '<img src="https://img.shields.io/badge/' . $block_text . '-' . $result_citation_count . '-blue.svg?style=flat" alt="' . t('Number of citations via Crossref') . ': ' . $result_citation_count . '">';
```

Comment:
@bondjimbond Not that I want to turn this into a huge thing, but perhaps you should use `format_string()` here to avoid the multiple concatenations? 
Something like...
```
$badge = format_string('<img src="https://img.shields.io/badge/!block_text-!result_citation_count-blue.svg?style=flat" alt="Number of citations via Crossref: !result_citation_count">',
  array(
    '!block_text' => $block_text,
    '!result_citation_count' => $result_citation_count,
  )
);
```

Reply:
Sure, that seems to work OK.